### PR TITLE
Feature/newsletter

### PR DESCRIPTION
**Pull request by TawKhoury**
**Table of Contents**

- [Description](#description)
- [Checklist](#checklist)

**Related issue(s):** #105

## Description
Newsletter component: 
* Newsletter and Newsletter Confirmation Components added to `htz-components`
* Newsletter colors and i18n added to `htz-theme`
* Graphql mutation Types and revolvers added in schema (`app-utils`)
* Newsletter signup added to createContext's posters (`htz-react-server`)

**Notes:**

## Checklist

Design:
  * [ ] Approved by designer

Usability:
  * [ ] Developer
  * [ ] Project manager

Tests
  * [ ] UI snapshot testing 
  * [ ] Unit testing

### CODE REVIEW

File: packages/components/htz-components/src/components/Newsletter/Newsletter.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+          loading={loading}
+          {...props}
+        />
+      )}
+    </Mutation>
+  );
+}
+
+// exporting NewsletterWithoutApollo for running tests without apollo client dependency.
+export { NewsletterWithoutApollo, };
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/Newsletter/Newsletter.js]: I'd move the unwrapped component to its own, separate file.

File: packages/apps/haaretz.co.il/components/Newsletter/Newsletter.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { Newsletter, } from '@haaretz/htz-components';
+
+export default Newsletter;
```
[the remark this answers is not in the document]
Comment (TawKhoury) [reply packages/apps/haaretz.co.il/components/Newsletter/Newsletter.js]: Removed

File: packages/apps/haaretz.co.il/components/Slot/Slot.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import dynamic from 'next/dynamic';
 import { AdSlot, } from '@haaretz/htz-components';
 
 import Comments from '../Comments/Comments';
+import Newsletter from '../Newsletter/Newsletter';
```
[the remark this answers is not in the document]
Comment (TawKhoury) [reply packages/apps/haaretz.co.il/components/Slot/Slot.js]: Done

File: packages/components/htz-components/src/components/Newsletter/Newsletter.js (hunk 1 above)
Comment (TawKhoury) [reply packages/components/htz-components/src/components/Newsletter/Newsletter.js]: Thanks for the review. All changes above has been fixed, committed and pushed.

**Pull request merged**